sp-files/.sphinx/requirements.txt

```diff
@@ -1,2 +1,2 @@
-canonical-sphinx
+git+https://github.com/canonical/canonical-sphinx@refs/pull/25/merge#egg=canonical-sphinx
```

Comment:
Now that https://github.com/canonical/canonical-sphinx-extensions/pull/41 is merged, do we still want to reference the PR?

Reply:
This one is for https://github.com/canonical/canonical-sphinx/pull/25 - but this has now been merged as well. :)

Reply:
Actually, it has been merged, but not made it into a release.
While we're developing this, it makes sense to get the latest version on main for the canonical-sphinx extension.
Updated.